### DISCUSSION TITLE
@click run twice

### DISCUSSION THREAD

**Discussion by getmonyinfo** [alpinejs/alpine#1897] [1. Help]
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>  
  <script  src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>


  <div x-cloak @notify="alert('hello')">
    <button @click.stop="$dispatch('notify')">Click me</button>
  </div>
</body>
</html>
```

**Comment by KevinBatdorf** [comment]
Can you set up a demo? For example, on codepen?

**Reply by getmonyinfo** [reply]
maybe no, because codepen can't set js order in html, 
`  <script  src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>` is at the top of **body** tag, **click** run twice,
but if the script is at the end of body, it's ok

I don't know why?

**Reply by KevinBatdorf** [reply]
What does your script look like? If you have the CDN, you don't want to use `Alpine.start()`

**Reply by getmonyinfo** [reply]
[@click run twice](https://www.w3schools.com/code/tryit.asp?filename=GT4IQ20FRR37)

**Reply by SimoTod** [reply]
Is there a reason why your alpine script is at the beginning of the body tag? The documentation says to add the tag in the head section with the defer attribute. Alternatively you can add it at the bottom of the body but if you put it at the top, Alpine will initialise when the dom is not ready and then will try to patch the dom while the browser builds in using its mutation observer.
